csharp/src/Google.Protobuf.Test/JsonParserTest.cs

```diff
@@ -584,8 +586,8 @@ public void NumberToFloat_Valid(string jsonValue, float expectedParsedValue)
         }
 
         [Test]
-        [TestCase("3.402824e38", typeof(InvalidProtocolBufferException))]
-        [TestCase("-3.402824e38", typeof(InvalidProtocolBufferException))]
+        [TestCase("3.4028236e38", typeof(InvalidProtocolBufferException))]
+        [TestCase("-3.4028236e38", typeof(InvalidProtocolBufferException))]
```

Comment:
super tiny nit (and just so that is clear at first sight that the string format is not the problem, use the same format in the valid and invalid tests):
```suggestion
        [TestCase("3.4028236e+38", typeof(InvalidProtocolBufferException))]
        [TestCase("-3.4028236e+38", typeof(InvalidProtocolBufferException))]
```

Reply:
Hmm... both formats should work. I'll update the tests to have both "with +" and without.